src/components/Articles/LikeArticle/LikeArticle.js

```diff
@@ -3,7 +3,7 @@ import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import { getArticleLikes, likeArticle, dislikeArticle } from '../../../actions';
-import { Button, LikeDislikeButton } from '../../common';
+import { LikeDislikeButton } from '../../common';
```

Comment:
how is this imported and not introduced in the code?

Reply:
ok i just forget to remove, i am going to remove it know